articles/firewall-manager/create-or-update-policyrules-azurepowershell

```diff
@@ -0,0 +1,152 @@
+
```

Comment:
```suggestion
title: 'Quickstart: Create an Azure Firewall policy using Azure PowerShell'
description: In this quickstart, you learn how create an Azure Firewall policy using Azure PowerShell.
services: firewall-manager
author: vaboya
ms.author: victorh
ms.date: 08/03/2021
ms.topic: quickstart
ms.service: firewall-manager
```

Reply:
@ktoliver OK, metadata block is in.

Reply:
@vidhoonv  the title should be "Azure Firewall Manager Policy update using Azure Powershell"
'manager ' is missing in the title section "title: 'Quickstart: Create an Azure Firewall policy using Azure PowerShell'"

Reply:
my bad @vhorne . It's azure firewall policy and not firewall manager.  Please help in correcting line number 65 by replacing to New-AzFirewallPolicy from New-AzFirewallManagerPolicy. Sorry about that.

Reply:
@vaboya I'll make those two revisions in the new pull request in the private repo.

Reply:
Oops, I neglected to commit the metadata block before moving the commit. I'll copy and paste into the new PR.